### ISSUE TITLE
Calibre ebook library manager "SBIE2205 Service not implemented" when using SBIE+ 1.4.2 & 1.5.1

### ISSUE THREAD

**Issue by ksmustang** [sandboxie-plus/Sandboxie#2418]
### Describe what you noticed and did

I use Calibre to manage my ebook library. It worked without problems in SBIE+ 1.0.20. However it started throwing "SBIE2205 Service not implemented" errors when i upgraded to 1.4.2 as well as to1.5.1

Calibre uses the Qt framework to render ebooks for reading or editing. The SBIE2205 errors occur when you attempt to read or edit any ebook in the library:

```
14:49:34.427   calibre-parallel.exe (10008): SBIE2205 Service not implemented:  CreateDesktop            
14:49:36.638   calibre-parallel.exe (7776): SBIE2205 Service not implemented:  ConsoleInit (C00000D4)    
14:49:36.674   calibre-parallel.exe (7588): SBIE2205 Service not implemented:  ConsoleInit (C00000D4)    
14:49:36.840   calibre-parallel.exe (10396): SBIE2205 Service not implemented:  ConsoleInit (C00000D4) 
```

 Additionally, Calibre displays the following message:
![C0AvAuZg](https://user-images.githubusercontent.com/12372772/198831124-d25f45ee-12a7-4c24-93da-daa1fa7d4882.jpeg)
`The Qt WebEngine Render process has crashed. You should try restarting the viewer.`

Therefore a change between Sandboxie-Plus 1.0.20 and 1.4.2 broke Calibre's ability to use the Qt framework to render ebooks.

As best I can tell, calibre.exe spawns calibre-parallel.exe which, in turn, initiates the Qt framework to display ebook contents.

### How often did you encounter it so far?

Every time I try to read or edit an ebook in versions 1.4.2 and 1.5.1

### Affected program

Calibre.exe

### Download link

https://calibre-ebook.com/download

### Where is the program located?

The program is installed only outside the sandbox.

### Expected behavior

The ebook should be presented in a readable book format (if reading), or as the ebook structure in html format along with a previewer (if editing)

### What is your Windows edition and version?

Windows 10 Pro 21H2 64-bit

### In which Windows account you have this problem?

Not relevant to my request.

### Please mention any installed security software

ESET Internet Security, but problem only occurred after upgrading SBIE+

### What version of Sandboxie are you running?

Sandboxie-Plus 1.5.1 64-bit

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### Is it a regression?

The problem observed after upgrading from 1.0.20 to 1.4.2 and 1.5.1

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on an empty sandbox?

I can confirm it also on an empty sandbox.

### Did you previously enable some security policy settings outside Sandboxie?

No

### Trace log

https://www.dropbox.com/s/aqfmo710jm144s8/Calibre.zip?dl=0

### Sandboxie.ini configuration

```shell
#
# Sandboxie configuration file
#

[GlobalSettings]

[UserSettings_052E014C]
SbieCtrl_AutoStartAgent=SandMan.exe

[DefaultBox]
Enabled=y
BlockNetworkFiles=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#00FFFF,ttl
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=AutoRecoverIgnore
ConfigLevel=9

[Calibre]
Enabled=y
ConfigLevel=9
BoxNameTitle=n
BorderColor=#00ffff,ttl,6

Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=AutoRecoverIgnore

AutoRecover=n
CopyLimitKb=81920
AutoDelete=y

ForceProcess=calibre.exe

NormalFilePath=C:\Users\ret\AppData\Roaming\calibre
NormalFilePath=C:\Users\ret\AppData\Local\Temp
NormalFilePath=E:\Users\ret\Documents

FileTrace=*
PipeTrace=*
KeyTrace=*
IpcTrace=*
GuiTrace=*
ClsidTrace=*
DebugTrace=y
```

**Comment by isaak654** [comment]
What about adding `UseSbieDeskHack=y` in your Calibre sandbox?

**Comment by ksmustang** [comment]
@isaak654 

> What about adding `UseSbieDeskHack=y` in your Calibre sandbox?

That resolved the issue with Calibre rendering ebooks when reading or editing

The  "SBIE2205 Service not implemented:  CreateDesktop" message is no longer generated.
         
The "SBIE2205 Service not implemented:  ConsoleInit (C00000D4)" message is still generated, but does not appear to adversely affect reading / editing functionality.

**Comment by AP2020C** [comment]
I added UseSbieDeskHack=y but I still get Error 1314
